plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PointAnnotationManager.kt

```diff
@@ -1052,6 +1033,35 @@ class PointAnnotationManager(
   override fun createDragLayer(): SymbolLayer {
     return symbolLayer(dragLayerId, dragSourceId) {}
   }
+  /**
+   * The TextLineHeight property
+   *
+   * Text leading value for multi-line text.
+   */
+  @Deprecated(
+    "text-line-height property is now data driven, use `PointAnnotation.textLineHeight` instead.",
+    ReplaceWith("PointAnnotation.textLineHeight")
+  )
+  var textLineHeight: Double?
```

Comment:
@Chaoba now the text-line-height property is data-driven, and has been introduced in the `PointAnntation`, so I deprecated this property in the `PointAnnotationManager`.